<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serveur</class>
 <widget class="QWidget" name="Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonLancerServeur">
       <property name="text">
        <string>Lancer serveur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="minimum">
        <number>3000</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5555</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditLogs"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Serveur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
